templates/service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-headless
```

Comment:
drop -headless and just use the .Release.Name.  This will make it easier to manage when all of the types use the same name